src/cachemachine/automatedtellermanager.py

```diff
@@ -0,0 +1,47 @@
+"""Manager for all the tellers and their cache deposits."""
+
+__all__ = [
+    "AutomatedTellerManager",
+]
+
+from dataclasses import dataclass, field
+from typing import Dict, List
+
+from aiohttp import web
+from aiojobs import Scheduler, create_scheduler
+from aiojobs._job import Job
+
+from cachemachine.automatedteller import AutomatedTeller
+
+
+@dataclass
+class AutomatedTellerManager:
+    _tellers: Dict[str, AutomatedTeller] = field(default_factory=dict)
+    _jobs: Dict[str, Job] = field(default_factory=dict)
+    _scheduler: Scheduler = None
+
+    async def init(self, app: web.Application) -> None:
+        self._scheduler = await create_scheduler()
+
+    async def cleanup(self, app: web.Application) -> None:
+        await self._scheduler.close()
+
+    def get_teller(self, name: str) -> AutomatedTeller:
+        return self._tellers[name]
```

Comment:
Oh, I see, the handlers catch `KeyError`. That works, although I tend to prefer to explicitly throw exceptions I define myself because the handlers would catch *every* `KeyError` from Python, including potentially errors that have nothing to do with that teller not being known if there's any complex code under the hood, which can make later troubleshooting harder.